src/helpers/dateHelper.js

```diff
@@ -7,13 +7,11 @@ export function getDay(now = new Date()){
 }
 
 export function incrementDate(date = new Date()){
-    date.setDate(date.getDate()+1);
-    return date;
+    return new Date(date.getTime() + 24 * 60 * 60 * 1000);
```

Comment:
I believe this change would not work in cases of daylight savings time changes (where there's technically 25 hours in a day)

Reply:
I'll use the previous impl. 
Only thing is that for the hook to work this methods need to return a new Date object